app/views/assignments/_assignment_form_options.html.erb

```diff
@@ -1,11 +1,19 @@
-<dl class="form">
+<% title_errors = @assignment.errors.full_messages_for(:title).join(", ") %>
```

Comment:
@nwoodthorpe I'd love to extract this logic into a view model, would you mind taking a look into that?